variables/_opacities.pcss

```diff
@@ -0,0 +1,9 @@
+:root {
+	/* -----------------------------------------------------------------------------
+	 * Opacities
+	 * ----------------------------------------------------------------------------- */
+
+	--opacity-hover: 0.8;
```

Comment:
Wandering if we could be more specific to which hover/active/default are these referring to. Thinking of month & week view, their multiday and overlap implementation, I think we'll end up having different ones and the name of the variables will have to be self explanatory

Reply:
I was just looking and wondering if we should rethink naming of these vars :joy

Reply:
Sure, any suggestions on what these should be?

Reply:
If they are the default values - the ones used most (throughout the plugins), then I think they are fine (or go with something like `--opacity-base-hover` etc) and let the outliers have more descriptive names.

Reply:
You are using these particularly for the events bar view switcher? Do you feel these will be repeated in other elements?

Reply:
I don't know what else they could/would be used for, but currently these are used for svg icons, since we can't fill with an rgba value with opacity.

Reply:
Ok, what do you think about having the `icon` word as part of the var name. Or if you feel it's also gonna be used in links in general -> `link`?

Reply:
`icon` is ok with me.

Reply:
Thanks!

Reply:
I know we've stayed away from doing specific naming for our vars and what not, but I wonder if it actually makes sense for something like opacity (`--opacity-08`)?

Reply:
and if we change the value to `0.7` because the hover style changes?

Reply:
Then it doesn't work :D Carry on, it's consistent with what we've been doing!

Reply:
To be clear if you went that path, you'd straight up use that var where needed, but not a blocker at all, carry on.